src/assets/i18n/ar.json

```diff
@@ -194,5 +194,7 @@
     "tr_attendance_sheet_created_successfully":"Attendance Sheet Created Successfully*",
     "tr_excel_sheet":"Excel Sheet*",
     "tr_upload_excel_sheet":"Upload Excel Sheet*",
-    "tr_national_holidays_dates":"National Holidays Dates*"
+    "tr_national_holidays_dates":"National Holidays Dates*",
+    "tr_applies_to":"Applies to*",
```

Comment:
also please add translations to the de.json file

Reply:
there is no de file in finger print ar,en and fr